ppmclibs/tinycv_impl.cc

```diff
@@ -41,22 +41,6 @@ using namespace std;
 
 struct Image {
     cv::Mat img;
-    mutable cv::Mat _preped;
-
-    cv::Mat prep() const
-    {
-        if (!_preped.empty() && false)
```

Comment:
I think this && false is the perfomance killer. And it slipped into a complete unrelated commit.

Because this prep is caching the blurred grayscale image in both the needle and the screen